stores/cartStore.ts

```
@@ -0,0 +1,541 @@
+    } catch (error) {
+      console.error('Update discount codes error:', error);
+      set({ 
+        error: error instanceof Error ? error.message : 'Failed to update discount codes',
+        isLoading: false 
+      });
+      throw error;
+    }
+  },
+
+  applyDiscountCode: async (code: string) => {
+  applyDiscountCode: async (code: string) => {
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)